gradle.properties

```diff
@@ -18,7 +18,6 @@ POM_DEVELOPER_NAME=RevenueCat, Inc.
 
 android.useAndroidX=true
 android.enableJetifier=true
-android.enableR8.libraries = false
```

Comment:
are we sure we no longer need this? do you remember why we had it in the first place?

Reply:
So I looked into it, and found where I added it https://github.com/RevenueCat/purchases-android/pull/72/files

I think the reason I added it is to not use R8 and use ProGuard since people were still using ProGuard. That way we would get errors when trying to build release. Enabling/Disabling R8 has been deprecated since, so I think we can remove this line. It would actually not compile with this line on.

<img width="949" alt="Screen Shot 2021-09-27 at 8 16 36 PM" src="https://user-images.githubusercontent.com/664544/135017088-f0fb582c-d1c9-4416-812e-23c426c5aa0d.png">

So I think if by default AS will use R8 now, we should just be using R8.